node26.cluster
/home/unibas/boittier/FDCM/fbuta_2/0_1
Traceback (most recent call last):
  File "/home/unibas/boittier/AdjustReference-System/ARS.py", line 436, in <module>
    print(f"Distance between Charge configurations = {ARS_obj.get_distance_charges()}")
  File "/home/unibas/boittier/AdjustReference-System/ARS.py", line 377, in get_distance_charges
    return Kabsch.align_vectors(self.c_positions_global, self.c_positions)[1]
  File "rotation.pyx", line 2250, in scipy.spatial.transform.rotation.Rotation.align_vectors
ValueError: Expected input `a` to have shape (N, 3), got ()
Distance between Atom configurations = 0.2622162099576393
cp: cannot stat ‘fbuta_2-0-1.xyz’: No such file or directory

real	0m0.425s
user	0m0.388s
sys	0m0.007s
-xyz
refined.xyz
-dens
/home/unibas/boittier/RDKit_G2/B.pdb/SCAN_1_2_3_4_S_36_10.0/1_SCAN_1_2_3_4_F/B.d.cube
-esp
/home/unibas/boittier/RDKit_G2/B.pdb/SCAN_1_2_3_4_S_36_10.0/1_SCAN_1_2_3_4_F/B.p.cube
-stepsize
0.2
-n_steps
100
-learning_rate
0.5
ERROR: Could not open "refined.xyz" for reading
cp: cannot stat ‘refined.xyz’: No such file or directory
/home/unibas/boittier/FDCM/fbuta_2/1_2
Traceback (most recent call last):
  File "/home/unibas/boittier/AdjustReference-System/ARS.py", line 432, in <module>
    ARS_obj = ARS(xyz_file_name, pcube, frame_file, pcube_2=pcube_2)
  File "/home/unibas/boittier/AdjustReference-System/ARS.py", line 167, in __init__
    self.c_positions, self.c_charges = read_mdcm_xyz(xyz_file_name)
  File "/home/unibas/boittier/AdjustReference-System/ARS.py", line 50, in read_mdcm_xyz
    xyz_file = open(filepath).readlines()
FileNotFoundError: [Errno 2] No such file or directory: '../0_1/1_final.xyz'
cp: cannot stat ‘fbuta_2-1-2.xyz’: No such file or directory

real	0m0.395s
user	0m0.385s
sys	0m0.009s
-xyz
refined.xyz
-dens
/home/unibas/boittier/RDKit_G2/B.pdb/SCAN_1_2_3_4_S_36_10.0/2_SCAN_1_2_3_4_F/B.d.cube
-esp
/home/unibas/boittier/RDKit_G2/B.pdb/SCAN_1_2_3_4_S_36_10.0/2_SCAN_1_2_3_4_F/B.p.cube
-stepsize
0.2
-n_steps
100
-learning_rate
0.5
ERROR: Could not open "refined.xyz" for reading
cp: cannot stat ‘refined.xyz’: No such file or directory
/home/unibas/boittier/FDCM/fbuta_2/2_3
Traceback (most recent call last):
  File "/home/unibas/boittier/AdjustReference-System/ARS.py", line 432, in <module>
    ARS_obj = ARS(xyz_file_name, pcube, frame_file, pcube_2=pcube_2)
  File "/home/unibas/boittier/AdjustReference-System/ARS.py", line 167, in __init__
    self.c_positions, self.c_charges = read_mdcm_xyz(xyz_file_name)
  File "/home/unibas/boittier/AdjustReference-System/ARS.py", line 50, in read_mdcm_xyz
    xyz_file = open(filepath).readlines()
FileNotFoundError: [Errno 2] No such file or directory: '../1_2/2_final.xyz'
cp: cannot stat ‘fbuta_2-2-3.xyz’: No such file or directory

real	0m0.391s
user	0m0.379s
sys	0m0.011s
-xyz
refined.xyz
-dens
/home/unibas/boittier/RDKit_G2/B.pdb/SCAN_1_2_3_4_S_36_10.0/3_SCAN_1_2_3_4_F/B.d.cube
-esp
/home/unibas/boittier/RDKit_G2/B.pdb/SCAN_1_2_3_4_S_36_10.0/3_SCAN_1_2_3_4_F/B.p.cube
-stepsize
0.2
-n_steps
100
-learning_rate
0.5
ERROR: Could not open "refined.xyz" for reading
cp: cannot stat ‘refined.xyz’: No such file or directory
/home/unibas/boittier/FDCM/fbuta_2/3_4
Traceback (most recent call last):
  File "/home/unibas/boittier/AdjustReference-System/ARS.py", line 432, in <module>
    ARS_obj = ARS(xyz_file_name, pcube, frame_file, pcube_2=pcube_2)
  File "/home/unibas/boittier/AdjustReference-System/ARS.py", line 167, in __init__
    self.c_positions, self.c_charges = read_mdcm_xyz(xyz_file_name)
  File "/home/unibas/boittier/AdjustReference-System/ARS.py", line 50, in read_mdcm_xyz
    xyz_file = open(filepath).readlines()
FileNotFoundError: [Errno 2] No such file or directory: '../2_3/3_final.xyz'
cp: cannot stat ‘fbuta_2-3-4.xyz’: No such file or directory

real	0m0.392s
user	0m0.382s
sys	0m0.009s
-xyz
refined.xyz
-dens
/home/unibas/boittier/RDKit_G2/B.pdb/SCAN_1_2_3_4_S_36_10.0/4_SCAN_1_2_3_4_F/B.d.cube
-esp
/home/unibas/boittier/RDKit_G2/B.pdb/SCAN_1_2_3_4_S_36_10.0/4_SCAN_1_2_3_4_F/B.p.cube
-stepsize
0.2
-n_steps
100
-learning_rate
0.5
ERROR: Could not open "refined.xyz" for reading
cp: cannot stat ‘refined.xyz’: No such file or directory
/home/unibas/boittier/FDCM/fbuta_2/4_5
slurmstepd: error: *** JOB 38322419 ON node26 CANCELLED AT 2021-10-01T10:51:06 ***
